src/Formatter/AirspaceFormatter.cpp

```diff
@@ -24,6 +24,33 @@ static const TCHAR *const airspace_class_names[] = {
   _T("Class G"),
   _T("Military Aerodrome Traffic Zone"),
   _T("Radio Mandatory Zone"),
+  _T("Unclassified"), // UNCLASSIFIED
+  _T("Restricted"),  // RESTRICTED
+  _T("TMA"),  // TMA
+  _T("Temporary Reserved Airspace"),  // TRA
+  _T("Temporary Segregated Area"),  // TSA
+  _T("Flight Information Region"),  // FIR
+  _T("Upper Flight Information Region"),  // UIR
+  _T("Air Defense Identification Zone"),  // ADIZ
+  _T("Aerodrome Traffic Zone"),  // ATZ
+  _T("Airway"),  // AWY
+  _T("Military Training Route"),  // MTR
+  _T("Alert Area"),  // ALERT
+  _T("Warning Area"),  // WARNING
+  _T("Protected Area"),  // PROTECTED
+  _T("Hazardous Area"),  // HTZ
+  _T("Gliding Sector"),  // GLIDING_SECTOR
+  _T("Temporary Reserved Prohibited Area"),  // TRP
+  _T("Terminal Information Zone"),  // TIZ
+  _T("Terminal Instrument Approach Procedure Area"),  // TIA
+  _T("Military Training Area"),  // MTA
+  _T("Control Area"),  // CTA
+  _T("Area Control Center Sector"),  // ACC_SECTOR
+  _T("Aerial Sporting Recreational"),  // AERIAL_SPORTING_RECREATIONAL
+  _T("Overflight Restriction"),  // OVERFLIGHT_RESTRICTION
+  _T("Military Restricted Area"),  // MRT
+  _T("Temporary Flight Restriction"),  // TFR
+  _T("Visual Flight Rules Sector"),  // VFR_SECTOR
```

Comment:
Where is authoritative documentation for these particular strings? There should be a comment with the URL to the document.

Reply:
Also, what do these obscure code comments mean?